resources/icons/20px/globe-network-add.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M12.75 7L12.7484 6.9903C12.8324 6.9968 12.9164 7 13 7C13 7.7545 13.2602 8.53866 13.8608 9.13921C13.9013 9.17968 13.9426 9.2186 13.9846 9.25601C13.9947 9.50104 14 9.7492 14 10C14 10.69 13.96 11.36 13.89 12H17.72C17.89 11.36 18 10.69 18 10C18 9.75343 17.986 9.50942 17.9601 9.26842C18.5763 8.74559 19 7.96496 19 7C19.1755 7 19.3517 6.98341 19.5261 6.9511C19.8338 7.91244 20 8.93693 20 10C20 15.52 15.52 20 10 20C4.48 20 0 15.52 0 10C0 4.48 4.48 0 10 0C11.0609 0 12.0835 0.165495 13.0431 0.472058C13.014 0.647857 13 0.824729 13 1C11.9515 1 11.1206 1.50027 10.6016 2.20416C10.4006 2.0705 10.1984 2 10 2C8.93 2 7.75 4.05 7.25 7H12.75ZM12.89 12C12.96 11.36 13 10.69 13 10C13 9.31 12.96 8.64 12.88 8H7.11C7.04 8.64 7 9.31 7 10C7 10.69 7.04 11.36 7.11 12H12.89ZM6.24 7C6.55 5.01 7.16 3.34 7.96 2.27C5.51 2.92 3.55 4.69 2.61 7H6.24ZM2 10C2 9.31 2.11 8.64 2.28 8H6.11C6.04 8.64 6 9.31 6 10C6 10.69 6.04 11.36 6.11 12H2.28C2.11 11.36 2 10.69 2 10ZM6.24 13H2.61C3.55 15.31 5.51 17.08 7.96 17.73C7.16 16.66 6.55 14.99 6.24 13ZM10 18C8.93 18 7.75 15.95 7.25 13H12.75C12.25 15.95 11.07 18 10 18ZM13.76 13C13.44 14.99 12.83 16.66 12.04 17.73C14.49 17.08 16.45 15.31 17.39 13H13.76Z" fill="#111418"/>
+<path d="M13 5C12.4 5 12 4.6 12 4C12 3.5 12.4 3 13 3H15V1C15 0.4 15.4 0 16 0C16.5 0 17 0.4 17 1V3H19C19.5 3 20 3.4 20 4C20 4.6 19.5 5 19 5H17V7C17 7.6 16.5 8 16 8C15.4 8 15 7.6 15 7V5H13Z" fill="#111418"/>
+</svg>
```

Comment:
Same for this one:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
  <path d="m12.75 7-.002-.01Q12.874 7 13 7c0 .755.26 1.539.86 2.14q.062.06.125.116A18 18 0 0 1 13.89 12h3.83c.17-.64.28-1.31.28-2q0-.37-.04-.732A2.92 2.92 0 0 0 19 7q.264 0 .526-.049C19.834 7.912 20 8.937 20 10c0 5.52-4.48 10-10 10S0 15.52 0 10 4.48 0 10 0c1.06 0 2.084.165 3.043.472Q13 .737 13 1c-1.049 0-1.88.5-2.398 1.204A1.1 1.1 0 0 0 10 2C8.93 2 7.75 4.05 7.25 7zm.14 5c.07-.64.11-1.31.11-2s-.04-1.36-.12-2H7.11a18.4 18.4 0 0 0 0 4zM6.24 7c.31-1.99.92-3.66 1.72-4.73A8 8 0 0 0 2.61 7zM2 10c0-.69.11-1.36.28-2h3.83a18.4 18.4 0 0 0 0 4H2.28c-.17-.64-.28-1.31-.28-2m4.24 3H2.61c.94 2.31 2.9 4.08 5.35 4.73-.8-1.07-1.41-2.74-1.72-4.73M10 18c-1.07 0-2.25-2.05-2.75-5h5.5c-.5 2.95-1.68 5-2.75 5m3.76-5c-.32 1.99-.93 3.66-1.72 4.73A8 8 0 0 0 17.39 13z"/>
  <path d="M13 5c-.6 0-1-.4-1-1 0-.5.4-1 1-1h2V1c0-.6.4-1 1-1 .5 0 1 .4 1 1v2h2c.5 0 1 .4 1 1s-.5 1-1 1h-2v2c0 .6-.5 1-1 1-.6 0-1-.4-1-1V5z"/>
</svg>

```

Reply:
Fixed! I had to keep `fill-rule="evenodd"` or I would get strange patterning in the icon
<img width="676" alt="Screenshot 2024-10-07 at 4 08 19 PM" src="https://github.com/user-attachments/assets/c227449b-5694-4516-a073-7c6716e19113">

Reply:
Ah, my bad! I neglected to visually inspect the path after optimizing the above. Modified the path in this commit to have it comply with the `nonzero` (default) rule: 401a01d3117c744ccfdae005f76ecd0d36e9c2f7